### PR TITLE
Adding PersistentRepository to registry.py

### PR DESCRIPTION
**Pull request by amanwriter**
PersistentRepository reads DB connection parameters from config.json in cwd and requires imports from psycopg2 and cauldron

### CODE REVIEW

**Comment by bhanuvrat** [comment]
@amanwriter please add cauldron to requirements/base.txt

File: vyked/registry.py [hunk 1]
```diff
@@ -128,17 +138,324 @@ def _get_full_service_name(service: str, version):
...
+        # for name, nodes in self._uptimes.items():
+        #     for node, uptimes in nodes.items():
+        #         if node == node_id:
+        #             uptimes['downtime'] = int(time.time())
+        return None
+
+    def get_uptimes(self):
+        #TODO: select * from uptimes
+        rows = self.qselect('uptimes', 'id', columns=['id', 'event_type', 'event_time'])
+        print("get_uptimes")
```
Comment (bhanuvrat) [inline vyked/registry.py]: @amanwriter get rid of print statements. Wherever necessary, write out to logs instead.

**Comment by kashifrazzaqui** [comment]
Also no commented code - delete it

**Comment by nerandell** [comment]
Outdated

**Pull request closed without merging**